modules/twitch_streaming

```diff
@@ -0,0 +1,56 @@
+
+"""
+Checks if a streamer is online. [Gross Gore/grossie_gore as placeholder]
+
+Checks if a streamer is online using the requests library to check to see
+if the dictionary key 'stream' in the json response is set to None or not.
```

Comment:
This is for the user.  They probably don't care how you did do this.  Maybe something like `queries the twitch kraken api to see if a chanel is streaming your favorite games.....`

Sorry I just skipped the line before as it was too long to read